guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -103,4 +103,4 @@
 .login-ui.continuation .login-dialog .logo,
 .login-ui.continuation .login-dialog .version {
     display: none;
-}
+}
```

Comment:
OK - something must be wrong with the way your changes were merged overall. I'll comment at the review level, but something is very fishy here.